features/compute-pressure.yml.dist

```diff
@@ -7,25 +7,14 @@ status:
     chrome: "125"
     edge: "125"
 compat_features:
-  # baseline: false
-  # support:
-  #   chrome: "125"
-  #   chrome_android: "125"
-  #   edge: "125"
+  - api.Permissions.permission_compute-pressure
```

Comment:
@foolip This block doesn't have a status now. What does that mean? That there's no support?

Reply:
Oh wait, all of the comments are removed. That's _very_ weird.

Reply:
Together with the other comment removed, it means that all of the keys have the same status as the overall feature.